ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3982,6 +3982,20 @@ AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC2X_VOADM2X_SAIDC2X_RAIDC2X_SSPC:
   - RAIDC2X
   - SSPC
 
+AHU_HTSC_VOADM_ETM_RMM_FDPM2X_RSM_SEFC_FAM_HRC:
```

Comment:
Please address abstract type recommendations above before reconstructing this canonical type.